scripts/buildMaterials.mjs

```
@@ -150,14 +168,26 @@ const generateComponents = () => {
 
         componentsMap.push({ component, npm })
 
+        const { package: packageName = '', version = '', exportName = '' } = npm || {}
+
         if (connection.connected) {
+          if (material.npm) {
+            // 给组件添加 npm 来源详细信息
+            const packageItem = packages.find((item) => item.package === packageName)
```

ESLint

[error] 'packages' is not defined.

(no-undef)